### DISCUSSION TITLE
New Endpoint inside swagger

### DISCUSSION THREAD

**Discussion by ryanachdiadsyah** [asternic/wuzapi#35] [Ideas]
In last update we have new endpoint inside [route.go](https://github.com/asternic/wuzapi/blob/main/routes.go) to create, read or delete new users before create a new session.

Look like this :
```
[GET] http://SERVER_URL:PORT/admin/users
[POST] http://SERVER_URL:PORT/admin/users
[DELETE] http://SERVER_URL:PORT/admin/users/{users_id}
```

the issue is we don't have that endpoint inside swagger ui.
can i add new endpoint to swagger, or leave it like that to hide this endpoint from swagger ui ?
Plz let me know which one you prefer

-happy coding

**Comment by asternic** [comment]
I am not sure if it should be there, as it does not share the authentication with the rest of endpoints? That is why I did not add it in the first place. Anyways, perhaps it should be added with a note that it will require the admin token instead of the regular user token. If you think its worthwhile, send the PR!

**Reply by ryanachdiadsyah** [reply]
I see what you mean.. i will test it in my local first, if is good for public i'll send a PR
